scene/main/viewport.cpp

```diff
@@ -2451,6 +2451,14 @@ void Viewport::_gui_update_mouse_over() {
 		return;
 	}
 
+	if (gui.sending_mouse_enter_exit_notifications) {
+		// If notifications are already being sent, delay call to next frame.
+		if (get_tree() && !get_tree()->is_connected("process_frame", callable_mp(this, &Viewport::_gui_update_mouse_over))) {
+			get_tree()->connect("process_frame", callable_mp(this, &Viewport::_gui_update_mouse_over), CONNECT_ONE_SHOT);
```

Comment:
Is is possible, that in a single Viewport this function could be connected to `process_frame` multiple times, so that the function is called more than once in the next frame?

Reply:
For this, I mostly copied what `editor/plugins/animation_player_editor_plugin.cpp:1707` did to connect the signal.
I check it's not already connected to make sure it won't be connected multiple times.
If the user makes multiple changes to top level or mouse filters, then this function would only need to be called once afterward.

Reply:
Sorry, I missed the check.